lib/src/misskey_chat.dart

```diff
@@ -0,0 +1,227 @@
+        "chat/messages/room-timeline", request.toJson());
+    return response.map((e) => ChatMessage.fromJson(e));
+  }
+
+  /// チャットメッセージを検索します
+  Future<Iterable<ChatMessage>> search(
+    ChatMessagesSearchRequest request,
+  ) async {
+    final response =
+        await _apiService.post<List>("chat/messages/search", request.toJson());
+    return response.map((e) => ChatMessage.fromJson(e));
+  }
+
+  /// チャットメッセージを取得します
+  Future<ChatMessage> show(ChatMessagesShowRequest request) async {
+    final response = await _apiService.post<Map<String, dynamic>>(
+        "chat/messages/show", request.toJson());
+    return ChatMessage.fromJson(response);
+  }
+
+  /// ユーザーのチャットを取得します
+  Future<Iterable<ChatMessage>> userTimeline(
+    ChatMessagesUserTimelineRequest request,
+  ) async {
+    final response = await _apiService.post<List>(
+        "chat/messages/user-timeline", request.toJson());
+    return response.map((e) => ChatMessage.fromJson(e));
+  }
+}
+
+class MisskeyChatRooms {
+  final ApiService _apiService;
+
+  final MisskeyChatRoomsInvitations invitations;
+
+  MisskeyChatRooms({required ApiService apiService})
+      : _apiService = apiService,
+        invitations = MisskeyChatRoomsInvitations(apiService: apiService);
+
+  /// チャットルームを作成します
+  Future<ChatRoom> create(ChatRoomsCreateRequest request) async {
+    final response = await _apiService.post<Map<String, dynamic>>(
+        "chat/rooms/create", request.toJson());
+    return ChatRoom.fromJson(response);
+  }
+
+  /// チャットルームを削除します
+  Future<void> delete(ChatRoomsDeleteRequest request) async {
+    await _apiService.post("chat/rooms/delete", request.toJson());
+  }
+
+  /// チャットルームに参加します
+  Future<void> join(ChatRoomsJoinRequest request) async {
+    await _apiService.post("chat/rooms/join", request.toJson());
+  }
+
+  /// 参加中のチャットルームを取得します
+  Future<List<ChatJoining>> joining(ChatRoomsJoiningRequest request) async {
+    final response =
+        await _apiService.post<List>("chat/rooms/joining", request.toJson());
+    return response.map((e) => ChatJoining.fromJson(e)).toList();
+  }
+
+  /// チャットルームを退室します
+  Future<void> leave(ChatRoomsLeaveRequest request) async {
+    await _apiService.post("chat/rooms/leave", request.toJson());
+  }
+
+  /// チャットルームのメンバーを取得します
+  Future<Iterable<ChatJoining>> members(ChatRoomsMembersRequest request) async {
+    final response =
+        await _apiService.post<List>("chat/rooms/members", request.toJson());
+    return response.map((e) => ChatJoining.fromJson(e));
+  }
+
+  /// チャットルームをミュートします
+  Future<void> mute(ChatRoomsMuteRequest request) async {
+    await _apiService.post("chat/rooms/mute", request.toJson());
+  }
+
+  /// 自身が所有しているチャットルームを取得します
+  Future<Iterable<ChatJoining>> owned(ChatRoomsOwnedRequest request) async {
+    final response =
+        await _apiService.post<List>("chat/rooms/owned", request.toJson());
+    return response.map((e) => ChatJoining.fromJson(e));
+  }
```

Comment:
```suggestion
  Future<Iterable<ChatRoom>> owned(ChatRoomsOwnedRequest request) async {
    final response =
        await _apiService.post<List>("chat/rooms/owned", request.toJson());
    return response.map((e) => ChatRoom.fromJson(e));
  }
```

ref: https://github.com/misskey-dev/misskey/blob/2025.4.0-beta.0/packages/backend/src/server/api/endpoints/chat/rooms/owned.ts#L20-L27